templates/addons/env/addons.parameters.yml

```diff
@@ -4,7 +4,5 @@
 # To be fixed under "DBTP-295 Only add the required parameters to addons.parameters.yml"
 Parameters:
   EnvironmentSecurityGroup: !Ref EnvironmentSecurityGroup
-  LBARN: !Ref PublicLoadBalancer
```

Comment:
Isn't `LBARN` used in `templates/addons/env/waf.yml`?

Reply:
Oh, but that's going away isn't it.

Reply:
Yes, it's dead in the water.  Jayesh is building the replacement